docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-infrastructure.adoc

```diff
@@ -132,200 +141,202 @@ config from cloud.resource where cloud.type = 'azure' AND api.name = 'azure-secu
 
-* `Microsoft.Synapse/workspaces/read`
-* `Microsoft.Insights/DiagnosticSettings/Read`
+* `Microsoft.MachineLearningServices/workspaces/read`
+* `Microsoft.MachineLearningServices/workspaces/computes/read`
 
 The Reader role includes the permissions. 
 
-|*Azure Relay*
-//RLP-134918
+|*Google Cloud Support*
+//RLP-131425
 
-|*azure-relay-namespaces*
+|*gcloud-cloud-support-case*
 
 Additional permission required:
 
-* `Microsoft.Relay/Namespaces/read`
-
-The Reader role includes the permission.
-
-|*Azure Relay*
-//RLP-134918
+* `cloudsupport.techCases.list`
 
-|*azure-relay-namespaces*
+The Viewer role includes the permission. 
 
-Additional permission required:
 
-* `Microsoft.Relay/Namespaces/read`
+|*Google Dataproc Clusters*
+//RLP-135383
 
-The Reader role includes the permission.
+|*gcloud-dataproc-cluster-job*
 
-|tt:[Update] *Azure Cognitive Services Account Diagnostic Settings*
+Additional permissions required:
 
-//RLP-137805
+* `dataproc.jobs.list` 
+* `dataproc.jobs.getIamPolicy`
 
-|*azure-cognitive-search-service-diagnostic-settings*
+The Viewer role includes the permissions. 
 
-Updated Asset Type for `azure-cognitive-search-service-diagnostic-settings` resources.
+|*Google Vertex AI AIPlatform*
+//RLP-135381
+|*gcloud-vertex-ai-aiplatform-feature-online-store*
 
-Previous name: "Azure Cognitive Services Account Diagnostic Settings"
+Additional permission required:
 
-New name: "Azure Cognitive Search Service Diagnostic Settings"
+* `aiplatform.featureOnlineStores.list`
 
-|*Google Vertex AI Platform*
-//RLP-136571
+The Viewer role includes the permission.
 
-|*gcloud-vertex-ai-aiplatform-notebook-runtime*
+|*Google Vertex AI AIPlatform*
+//RLP-135380
+|*gcloud-vertex-ai-aiplatform-feature-group*
 
 Additional permission required:
 
-* `aiplatform.notebookRuntimes.list`
+* `aiplatform.featureGroups.list`
 
-The Viewer role includes the permission. 
+The Viewer role includes the permission.
 
 |*Google Workspace*
 //RLP-135384
-
 |*gcloud-domain-user-last-login-time*
 
-No additional permissions are required.
-
-|*Google Cloud VMware Engine*
-//RLP-135383
-
-|*gcloud-vmware-engine-external-address*
-
-Additional permissions required:
-
-* `vmwareengine.locations.list`
-* `vmwareengine.privateClouds.list`
-* `vmwareengine.externalAddresses.list`
+No additional permissions required for this API.
+|===
 
-The Viewer role includes the permission.
+[#api-rate-limits]
+=== API Rate Limits
+//RLP-129569, RLP-139236
+Prisma Cloud uses API https://pan.dev/prisma-cloud/api/cspm/rate-limits/[rate limits] at the endpoint level to protect the performance and availability of its services. Rate limits will be applied to additional APIs starting from the 24.6.1 release. Details of these rate limits will be made available in the supporting API documentation and Look Ahead Notices when these limits are introduced. 
 
-|*Google Vertex AI Platform*
-//RLP-135381
+[cols="60%a,20%a,20%a"]
+|===
+|*Endpoint*
+|*Rate Limit*
+|*Burst Rate*
```

Comment:
1. Update table headers:  Rate Limit and Burst Rate  change to ->  `Rate Limit (tps)` and `Burst Rate (tps)`
2. Change sentence to `Rate limits will apply to the following endpoints in release 24.6.1:`


_snippet of table header changes_
```
[#api-rate-limits]
=== API Rate Limits 
//RLP-129569, RLP-139236

Rate limits will apply to the following endpoints in release `24.6.1`:

[cols="60%a,20%a,20%a"]
|===
|*Endpoint*
|*Rate Limit (tps)*
|*Burst Rate (tps)*
```

![image](https://github.com/hlxsites/prisma-cloud-docs/assets/3386501/e95b8939-b2c0-4182-bd57-2c026994f8d1)